src/App.vue

```diff
@@ -674,10 +674,11 @@ export default {
     },
 
     storeNrData (nr: NameRequestIF, ia: any): void {
-      // check if NR is valid
-      if (!this.isNrValid(nr)) {
+      // verify that NR is valid
+      const error = this.isNrInvalid(nr)
+      if (error) {
         this.nameRequestInvalidDialog = true
-        throw new Error('Invalid NR data')
+        throw new Error(error)
```

Comment:
This won't throw "true" because error is actually a string. The condition on line 679 passes if the string is truthy or fails if error is empty/null/undefined.